scripts/centos/vmware.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh -eux
```

Comment:
Good idea! changed it. See if this is what you have in mind.

Note 1: Although it is possible to select which script to execute in the packer file, it seemed like a lot of json just to get rid of a couple of "cases" so a left them on the scripts

Note 2: I don't have access to parallels so I can not test it. The change is pretty simple so I believe that it should work.